console.go

```diff
@@ -22,6 +22,10 @@ func (l *writerLogger) Writer(sev Severity) io.Writer {
 	return nil
 }
 
+func (l *writerLogger) SetSeverity(sev Severity) {
+	l.sev = sev
```

Comment:
I don't think a note, nor a atomic write is needed in this case.

strictly speaking, totally, yes. but in reality, a write to an int32 doesn't matter, and making all CPUs see the write at the same time (eg, actually atomic), doesn't actually matter.  we are flipping it from values of 0, 1, 2, 3.